composer.json

```diff
@@ -11,10 +11,9 @@
     "license": "MIT",
     "require": {
         "php": ">=7.4",
-        "illuminate/support": "^8.0|^9.0|^10.0",
-        "illuminate/console": "^8.0|^9.0|^10.0",
-        "illuminate/filesystem": "^8.0|^9.0|^10.0",
-        "doctrine/dbal": "^2.9|^2.10|^3.0"
```

Comment:
@demianottema can you re-add `doctrine/dbal` to the list of dependencies?
It may not be required by Laravel 11.x but I want to keep the package backwards-compatible.